bin/install

```diff
@@ -1,4 +1,4 @@
-#!/bin/env bash
+#!/usr/bin/env bash
```

Comment:
This is the same shebang used in `asdf` itself.

Reply:
Exactly, which is correct. Here an excerpt from https://www.in-ulm.de/~mascheck/various/shebang/#env

> However, the location of env(1) might vary. Free-, Net-, OpenBSD and some Linux distributions (e.g. Debian) only come with /usr/bin/env.
On the other hand, there's only /bin/env at least on SCO OpenServer 5.0.6 and Cray Unicos 9.0.2 (although the latter is only of historical interest).
On some other Linux distributions (Redhat) it's located in /bin and /usr/bin/ contains a symbolic link pointing to it.
>
> The env-mechanism is highly increasing convenience, and almost all systems nowadays provide /usr/bin/env. Yet, it cannot strictly assure "portability" of a script.

Reply:
This was an oversight. Thanks!